### PR TITLE
Improve ci build

### PR DESCRIPTION
**Pull request by MagicMarvMan**
Hello @niccokunzmann .

I hope you are doing well.

I improved the CI build a little bit by outsourcing it to a script and adding a html test.

Thank you,
MagicMarvMan

### CODE REVIEW

File: _includes/stylesheets.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 <!-- Start stylesheets -->
-<link type="text/css" rel="stylesheet" href="{{ '/styles/preloader.css' | relative_url }}{% if site.github.build_revision %}?v={{ site.github.build_revision }}{% endif %}">
-<link type="text/css" rel="stylesheet" href="{{ '/styles/styles.css' | relative_url }}{% if site.github.build_revision %}?v={{ site.github.build_revision }}{% endif %}">
-<link type="text/css" rel="stylesheet" href="{{ '/styles/font-awesome.min.css' | relative_url }}{% if site.github.build_revision %}?v={{ site.github.build_revision }}{% endif %}">
+<link type="text/css" rel="stylesheet" href="{{ '/styles/preloader.css' | relative_url }}{% if site.github.build_revision %}?v={{ site.github.build_revision }}{% endif %}" data-proofer-ignore>
```
Comment (niccokunzmann) [inline _includes/stylesheets.html]: Why do we need to add `data-proofer-ignore`?

File: script/cibuild [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+bundle exec jekyll build
```
Comment (niccokunzmann) [inline script/cibuild]: yes, good idea ... Should we also add the Gemfile.lock, so the build is reproducible?

File: script/cibuild [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+bundle exec htmlproofer ./_site --disable-external
```
Comment (niccokunzmann) [inline script/cibuild]: Is there a build where I can see what happens if `data-proofer-ignore` is not present?

**Comment by niccokunzmann** [comment]
I see two test failing:

1. the urls https://travis-ci.com/Jugendhackt/Handbuch-Jugend-Hackathons/builds/77153269#L681 `internal script /Handbuch-Jugend-Hackathons/js/lib/angular-touch.min.js does not exist (line 2960)`
2. `internally linking to appendix/09 Zeitplan Veranstaltungsvorbereitung.ods, which does not exist (line 42)`

**Comment by niccokunzmann** [comment]
Ignoring the errors is possible but is it also possible to correct them? What to they mean?

File: _includes/stylesheets.html (hunk 1 above)
Comment (MagicMarvMan) [reply _includes/stylesheets.html]: If we don't use `data-proofer-ignore`, html-proofer tries to find the stylesheets but can't find them as the url is unknown to proofer.

File: script/cibuild (hunk 2 above)
Comment (MagicMarvMan) [reply script/cibuild]: Yes, good idea. I will add it.

**Comment by niccokunzmann** [comment]
Okay, thanks!

**Pull request merged**